fixture-generator/Makefile

```
@@ -0,0 +1,60 @@
+# Kibana Fixture Generator
+# Generates test fixtures using Kibana's LensConfigBuilder API
+
+IMAGE_NAME := kibana-fixture-generator
+KIBANA_VERSION ?= v9.2.0
+OUTPUT_DIR := $(shell pwd)/output
+
+.PHONY: build run shell clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)